xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -119,8 +135,13 @@ public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
     lookasideChannelLb.handleResolvedAddresses(resolvedAddresses);
   }
 
+  /**
+   * Creates a balancer that is associated to a balancer name. The is to handle the legacy usecase
+   * that requires balancerName from xds config.
+   */
   private LoadBalancerProvider newLookasideChannelLbProvider(
```

Comment:
Why this needs to be a method? Basically it only creates a new object and return it? Why wrap this into a method? I would say you should clean things up as you move them around. Otherwise, this sort of things will spread everywhere.

Reply:
Cleaned up the method.